src/bolero/support.rs

```diff
@@ -348,15 +348,18 @@ impl ValueGenerator for UniqueV6InterfaceAddressGenerator {
 pub const ALPHA_NUMERIC_CHARS: &str =
     "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
 
-pub struct MacAddrString(pub String);
+pub struct UnicastMacAddrString(pub String);
```

Comment:
I would recommend against making this field `pub`.

It allows easy violation of the type contract.

Reply:
nit: what we really want here is something which maps to `SourceMac` as defined in https://github.com/githedgehog/dataplane/blob/4b631ef1e13c01429c1ec5cc5b12de59a93e9f24/net/src/eth/mac.rs#L197-L204

More specifically, here: https://github.com/githedgehog/dataplane/blob/4b631ef1e13c01429c1ec5cc5b12de59a93e9f24/net/src/eth/mac.rs#L140-L153

Zero is also not legal.

That said, this is 100% better than what was here so not blocking